rocketpy/motors/LiquidMotor.py

```diff
@@ -1,365 +1,365 @@
-        print("Total Burning Time: " + str(self.burnOutTime) + " s")
-        print(
-            "Total Propellant Mass: "
-            + "{:.3f}".format(self.propellantInitialMass)
-            + " kg"
-        )
-        print(
-            "Propellant Exhaust Velocity: "
-            + "{:.3f}".format(self.exhaustVelocity)
-            + " m/s"
-        )
-        print("Average Thrust: " + "{:.3f}".format(self.averageThrust) + " N")
-        print(
-            "Maximum Thrust: "
-            + str(self.maxThrust)
-            + " N at "
-            + str(self.maxThrustTime)
-            + " s after ignition."
-        )
-        print("Total Impulse: " + "{:.3f}".format(self.totalImpulse) + " Ns")
-
-        # Show plots
-        print("\nPlots")
-        self.thrust.plot(0, self.burnOutTime)
-        self.mass.plot(0, self.burnOutTime)
-        self.massFlowRate.plot(0, self.burnOutTime)
-        self.centerOfMass.plot(0, self.burnOutTime, samples=50)
-        self.inertiaTensor[0].plot(0, self.burnOutTime, samples=50)
-        self.inertiaTensor[2].plot(0, self.burnOutTime, samples=50)
+# -*- coding: utf-8 -*-
+
+__author__ = "Giovani Hidalgo Ceotto, Oscar Mauricio Prada Ramirez, João Lemes Gribel Soares, Mateus Stano, Pedro Henrique Marinho Bressan, Patrick Bales, Lakshman Peri, Gautam Yarramreddy, Curtis Hu, and William Bradford"
+__copyright__ = "Copyright 20XX, RocketPy Team"
+__license__ = "MIT"
+
+import numpy as np
+from functools import cached_property
+
+from rocketpy.motors import Motor
+from rocketpy.Function import funcify_method, Function
+
+
+class LiquidMotor(Motor):
+    """Class to specify characteristics and useful operations for Liquid
+    motors."""
+
+    def __init__(
+        self,
+        thrustSource,
+        nozzleRadius,
+        burn_time=None,
+        nozzlePosition=0,
+        reshapeThrustCurve=False,
+        interpolationMethod="linear",
+        coordinateSystemOrientation="nozzleToCombustionChamber",
+    ):
+        """Initialize Motor class, process thrust curve and geometrical
+        parameters and store results.
+
+        Parameters
+        ----------
+        thrustSource : int, float, callable, string, array
+            Motor's thrust curve. Can be given as an int or float, in which
+            case the thrust will be considered constant in time. It can
+            also be given as a callable function, whose argument is time in
+            seconds and returns the thrust supplied by the motor in the
+            instant. If a string is given, it must point to a .csv or .eng file.
+            The .csv file shall contain no headers and the first column must
+            specify time in seconds, while the second column specifies thrust.
+            Arrays may also be specified, following rules set by the class
+            Function. See help(Function). Thrust units are Newtons.
+        burnOut : int, float
+            Motor burn out time in seconds.
+        nozzleRadius : int, float

```

Comment:
Need to change the docs here, and probably in all motor classes

Reply:
Solved by [081f0de](https://github.com/RocketPy-Team/RocketPy/pull/337/commits/081f0de83c750b1e936a149b49759e5f97e8ea96).